File: seminar_1/Python/task3.py
# 3. Напишите программу, которая будет выдавать название дня недели по заданному номеру.
# 3 -> Среда
# 5 -> Пятница

# Версия через условия
week_num = int(input("Введите номер дня недели: "))
if 0 < week_num < 8:
    if week_num == 1:
        print("Понедельник")
    elif week_num == 2:
        print("Вторник")
    elif week_num == 3:
        print("Среда")
    elif week_num == 4:
        print("Четверг")
    elif week_num == 5:
        print("Пятница")
    elif week_num == 6:
        print("Суббота")
    elif week_num == 7:
        print("Воскресенье")
else:
    print("Такого дня в неделе нет")


# Версия через массив
week = ["Понедельник", "Вторник", "Среда",
        "Четверг", "Пятница", "Суббота", "Воскресенье"]
week_num = int(input("Введите номер дня недели: "))
if 0 < week_num < 8:
    print(week[week_num - 1])
else:
    print("Такого дня в неделе нет")

File: seminar_1/Java/task3.java
// 3. Напишите программу, которая будет выдавать название дня недели по заданному номеру.
// 3 -> Среда
// 5 -> Пятница
package seminar_1.Java;

import java.util.Scanner;

public class task3 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String day = input.nextLine();
        switch (day) {
            case "1" -> System.out.println("Понедельник");
            case "2" -> System.out.println("Вторник");
            case "3" -> System.out.println("Среда");
            case "4" -> System.out.println("Четверг");
            case "5" -> System.out.println("Пятница");
            case "6" -> System.out.println("Суббота");
            case "7" -> System.out.println("Воскресенье");
            default -> System.out.println("Не день недели");
        }
        input.close();
    }
}
